app/views/users/show.html.erb

```diff
@@ -1,3 +1,24 @@
-<h1>Users#show</h1>
-<p>Here are the informations of a given user</p>
-# <p>Find me in app/views/users/show.html.erb</p>
+<div class="container max-w-screen-md m-auto">
+  <%= render partial: "shared/user_card_header", locals: { user: @user } %>
+  <div class="font-mono shadow-lg border-b-4 border-slate-600 p-4 -mt-8">
+    <h2 class="font-bold text-3xl">Bio</h2>
+    <p><%= @user.bio %></p>
+  </div>
+  <div class="font-mono border-b-2 mt-14 mb-8">
+    <% if @posts.any? %>
+      <div class="grid grid-cols-1">
+        <% @posts.each do |post| %>
+          <%= link_to user_post_path(@user, post) do %>
+            <%= render partial: "shared/post_card_preview", locals: { post: post } %>
+          <% end %>
+        <% end %>
+      </div>
+      <div class="w-fit mx-auto my-8">
+        <%= link_to "See all posts" , user_posts_path(@user),
```

Comment:
- [ ] Hello, @lily4178993! 👋 Great job on implementing the link that navigates to the post#index of a specific user. 🌟 However, I've noticed a slight deviation from the project requirements. 🚧 In your current implementation, when I visit this path, I can only see the three most recent posts. 📆 This isn't the intended behavior; the path should display all posts for the given user. 📚 I've attached an image to help you visualize the expected structure of this page. 🖼️ Let's make sure it aligns with the project requirements! 🤝

![postindex](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAACLTKYWCR7BEWLLAVH2NW6WCZL4PPXQ)

Reply:
Hi @zeff96 ,
Thank you for the review and feedback!

Reply:
- Well resolved 👍🏽